src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMarginViewModel.cs

```diff
@@ -60,13 +62,20 @@ internal class InheritanceMarginViewModel
             if (members.Length == 1)
             {
                 var member = tag.MembersOnLine[0];
-                var textAppended = " " + EditorFeaturesWpfResources.is_inherited;
+                // var textAppended = " " + ServicesVSResources.is_inherited;
+
+                // Here we want to show a classified text with loc text,
+                // e.g. 'Bar' is inherited.
+                // But the classified text are inlines, so can't directly use string.format to generate the string
                 var inlines = member.DisplayTexts.ToInlines(classificationFormatMap, classificationTypeMap);
-                WrapMemberWithinApostrophe(inlines);
-                inlines.Add(new Run(textAppended));
+                var startOfThePlaceholder = ServicesVSResources._0_is_inherited.IndexOf("{0}", StringComparison.Ordinal);
+                var prefixString = ServicesVSResources._0_is_inherited[..startOfThePlaceholder];
+                var suffixString = ServicesVSResources._0_is_inherited[(startOfThePlaceholder + "{0}".Length)..];
+                inlines.Insert(0, new Run(prefixString));
+                inlines.Add(new Run(suffixString));
                 var toolTipTextBlock = inlines.ToTextBlock(classificationFormatMap);
```

Comment:
you may need to isnert a ltr marker to ensure that our code shows up property left-to-right, even when inserted into an rtl language.

Reply:
Good to know new things in WPF : )